concepticondata/concepticon.tsv

```diff
@@ -416,19 +416,19 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 415	CARRY UNDER ARM	Motion	To hold or to take something from one place to another under the arm.	Action/Process	
 416	OUTRIGGER	Motion	A stabilizer for a canoe; spars attach to a shaped log or float parallel to the hull.	Person/Thing	
 417	THREE TIMES	Quantity	Three times.	Other	
-418	FOR A LONG TIME	Time	Great amount of time between two specific events.	Other	
+418	FOR A LONG TIME	Time	(Of an action or state) to last a long period of time.	Property	
 419	SEVERE	Miscellaneous function words	Intensely or extremely bad or unpleasant in degree or quality.	Property	
 420	FOREST	The physical world	A vegetation community dominated by trees and other woody shrubs, growing close enough together that the tree tops touch or overlap, creating various degrees of shade on the forest floor.	Person/Thing	
 421	RETREAT	Warfare and hunting	To retreat unwillingly and/or suddenly, often caused by sudden pain, shock, scare, fright, surprise, or similar.	Action/Process	
 422	NECESSITY	Cognition	Something that needs to be done.	Person/Thing	
 423	GET LOST	Basic actions and technology	To end up in an unknown place, to be not to be found again, to get lost, to irrecoverably slip away.	Action/Process	
-424	MARRIED MAN	Kinship	A married man	Person/Thing	
-425	BE LATE	Time	to be late	Action/Process	
-426	MARRIED WOMAN	Kinship	a married woman	Person/Thing	
+424	MARRIED MAN	Kinship	A married man.	Person/Thing	
+425	BE LATE	Time	To be later than the expected time.	Property	
```

Comment:
Are we sure about this being a property and not a verb? In Portuguese we have almost that, it is always in an inchoative aspect but it is a verb.

Reply:
Seems a bit like the issue with "to be alive" (which currently is as "action/process" but which I feel might also rather be "property"). It's about the meaning, not about how it's expressed in a given language after all.

Reply:
I'd still guess that the more likely elicited answer is a verb than a noun (and this considering that languages really distinguish them and, especially, that the researcher doesn't strip all the morphology), but without any additional material your guess is as good as mine.

So, for consistency, let's keep what you are proposing. We can always change later if needed.